core/app/c/[communitySlug]/types/page.tsx

```diff
@@ -48,7 +48,10 @@ export default async function Page({
 
 	const [types, { fields }] = await Promise.all([
 		getAllPubTypesForCommunity(communitySlug).execute(),
-		getPubFields({ communityId: community.id }).executeTakeFirstOrThrow(),
+		getPubFields({
+			communityId: community.id,
+			includeRelations: true,
```

Comment:
I had added `includeRelations` while the relationship stuff was still being ironed out and we didn't want it showing everywhere. if we are always showing relations for every query now, then we might as well remove the lil bit of code that `includeRelations` triggers and always return as default!

I just looked through the code base and there are a few places we aren't passing `includeRelations` though I don't know if this is intentional. anyway, not important for this PR but for the future it's a potential thing to clean up!